schema/deploy/shipping/views.sql

```diff
@@ -727,7 +727,6 @@ create or replace view shipping.sample_with_best_available_encounter_data_v1 as
   left join samples_with_manifest_data using (sample_id)
   left join site_details on (site_manifest_details similar to manifest_regex)
   left join warehouse.site on (samples_with_manifest_data.site_id = site.site_id)
-  where sample.identifier is not null
```

Comment:
I wanted to start a separate thread on this filter removal:

Just wanted to confirm that by removing this, we're increasing both the number of samples with and without encounter data returned by this view. Is that desirable?

Reply:
I'm not sure what you mean. I thought this view was to provide the best available metadata for all samples in `warehouse.samples`

Reply:
Ok, that seems fine to me.